modules/tests/shared/src/test/scala/lucuma/core/math/UnitsSuite.scala

```diff
@@ -16,33 +18,33 @@ final class UnitsSuite extends FunSuite {
 
   test("Sanity tests") {
     assert(Rational.one.withUnit[Angstrom] === Rational(1, SafeLong(10).pow(10)).withUnit[Meter])
```

Comment:
@erikerlandson currently getting a compile error for this:
```
[error] -- Error: /workspace/lucuma-core/modules/tests/shared/src/test/scala/lucuma/core/math/UnitsSuite.scala:20:96 
[error] 20 |    assert(Rational.one.withUnit[Angstrom] === Rational(1, SafeLong(10).pow(10)).withUnit[Meter])
[error]    |                                                                                                ^
[error]    |                             unit type Meter not convertable to Angstrom
[error]    | This location contains code that was inlined from UnitsSuite.scala:20
[error]    | This location contains code that was inlined from unit.scala:27
[error]    | This location contains code that was inlined from policy.scala:23
[error] one error found
```

`Angstrom` is defined here:
https://github.com/gemini-hlsw/lucuma-core/blob/f6da50b2326b45448a8cd7c7a966333b8e68f09f/modules/core/shared/src/main/scala/lucuma/core/math/units.scala#L31-L32

I assume it's a simple and stupid mistake 😅 thanks!

Reply:
@armanbilge   my guess is this:
`import coulomb.units.si._`

should be `import coulomb.units.si.{*, given}`

In scala 3 givens aren't automatically imported via `*` or I assume `_`

Reply:
Right, I've been trying to fix those. But is that import needed at the definition-site or the use-site?

Reply:
it should not be needed for the definition

Reply:
but it is  needed here:
```
assert(Rational.one.withUnit[Angstrom] === Rational(1, SafeLong(10).pow(10)).withUnit[Meter])
```

because `===` needs to instantiate the conversion, and that requires the `given`s

Reply:
You need to import `given` here too:

https://github.com/gemini-hlsw/lucuma-core/blob/f6da50b2326b45448a8cd7c7a966333b8e68f09f/modules/tests/shared/src/test/scala/lucuma/core/math/UnitsSuite.scala#L11

Reply:
I think you're looking at an old commit, see https://github.com/gemini-hlsw/lucuma-core/pull/520#discussion_r869613369.

Reply:
also, is this supposed to be `trait` ?
https://github.com/gemini-hlsw/lucuma-core/blob/f6da50b2326b45448a8cd7c7a966333b8e68f09f/modules/core/shared/src/main/scala/lucuma/core/math/units.scala#L22

Reply:
Yes. Then it is inherited by an `object`.